lib/views/pr-review-comments-view.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Point, Range} from 'atom';
+
+import {toNativePathSep} from '../helpers';
+import Marker from '../atom/marker';
+import Decoration from '../atom/decoration';
+import GithubDotcomMarkdown from './github-dotcom-markdown';
+import Timeago from './timeago';
+
+export default class PullRequestCommentsView extends React.Component {
+  static propTypes = {
+    relay: PropTypes.shape({
+      hasMore: PropTypes.func.isRequired,
+      loadMore: PropTypes.func.isRequired,
+      isLoading: PropTypes.func.isRequired,
+    }).isRequired,
+    commentThreads: PropTypes.arrayOf(PropTypes.shape({
+      rootCommentId: PropTypes.string.isRequired,
+      comments: PropTypes.arrayOf(PropTypes.object).isRequired,
+    })),
+    getBufferRowForDiffPosition: PropTypes.func.isRequired,
+    switchToIssueish: PropTypes.func.isRequired,
+  }
+
+  render() {
+    return [...this.props.commentThreads].map(({rootCommentId, comments}) => {
+      const rootComment = comments[0];
+      if (!rootComment.position) {
+        return null;
+      }
+
+      const nativePath = toNativePathSep(rootComment.path);
+      const row = this.props.getBufferRowForDiffPosition(nativePath, rootComment.position);
+      const point = new Point(row, 0);
+      const range = new Range(point, point);
```

Comment:
We could probably extract a helper for "make a one-point `Range` at position 0 of this row". 🤔

Reply:
Hmm, this is the only place we're doing that though. So not sure we need a helper.